doc/rules/phpdoc/phpdoc_types.rst

```diff
@@ -12,9 +12,9 @@ Configuration
 
 Type groups to fix.
 
-Allowed values: a subset of ``['alias', 'meta', 'simple']``
```

Comment:
Even the old values are not easy to understand, what about deprecating `groups` and by default fixing all, but add a new option `exclude`, so people can exclude... well `resource`?

//cc @keradus